### DISCUSSION TITLE
Add Pinniped as Ory Adopter

### DISCUSSION THREAD

**Discussion by vinckr** [vmware-tanzu/pinniped#1334] [Ideas] [answered]
Hello folks, 
I saw that [pinniped](https://github.com/vmware-tanzu/pinniped/network/dependencies) is using a number of Ory products (ory/fosite being the main one probably). 
Would you be open to be added to the [official Ory adopter list](https://github.com/ory/meta/blob/master/templates/repository/common/ADOPTERS.md)?

I am happy to make the change, just wanted to give a heads up. 
I think this could be a the start of a great partnership 🤝

**Comment by cfryanr** [comment, marked as answer]
Hi @vinckr,

Here at the open source Pinniped project, we're fans of ory/fosite and we use it as a library in the production code of the Pinniped Supervisor component quite extensively. It's been great!

I think the open source Pinniped project (https://pinniped.dev) would be happy to be listed as a user of the fosite library. However, "Tanzu" is a brand name for a family of products and services from VMware, so I can't personally speak on behalf of all of Tanzu. If you'd like to list Pinniped (https://pinniped.dev) as a user of fosite, please feel free.

**Reply by vinckr** [reply]
That makes sense! Will follow up with the PR here once I get to it, 
thanks

**Reply by vinckr** [reply]
Hello @cfryanr 
Apologies for the late followup, but do you happen to have the pinniped logo available as vector graphic somewhere? 
I figured I found one [here](https://pinniped.dev/img/logo.svg), but turns out that is just an svg containing a "normal" png.

Thanks!

**Reply by joshuatcasey** [reply]
@vinckr Let me know if this works for you.

https://drive.google.com/file/d/1AscaELvPWv4-imXLQikg7KwxvdF0FfMp/view?usp=sharing

**Reply by vinckr** [reply]
@joshuatcasey 
Perfect!
[PR](https://github.com/ory/meta/pull/207) is merged, you will be added to the list with the next repo synch action. 

Thanks! 

PS: let me know if you want to publish something pinniped/ory related on our [Blog](https://www.ory.sh/blog/) or if speaking on our [Ory dev conference](https://www.ory.sh/summit/2022/) would be of any interest to you :-)
